charts/vector/values.yaml

```diff
@@ -343,105 +342,102 @@ dnsConfig: {}
+  # podMonitor.honorLabels -- If true, honor_labels is set to true in the [scrape config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config).
   honorLabels: false
-  # podMonitor.honorTimestamps -- If true, honor_timestamps is set to true in scrape config
-  ## Ref: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config
+  # podMonitor.honorTimestamps -- If true, honor_timestamps is set to true in the [scrape config](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config).
   honorTimestamps: true
 
-## Optional built-in HAProxy load balancer
+# Optional built-in HAProxy load.
 haproxy:
-  # haproxy.enabled -- If true, create a HAProxy load balancer
+  # haproxy.enabled -- If true, create a HAProxy load balancer.
   enabled: false
 
-  ## Define the HAProxy image to use
+  # Define the HAProxy image to use.
   image:
-    # haproxy.image.repository -- Override default registry + name for HAProxy
+    # haproxy.image.repository -- Override default registry and name for HAProxy.
     repository: haproxytech/haproxy-alpine
-    # haproxy.image.pullPolicy -- HAProxy image pullPolicy
+    # haproxy.image.pullPolicy -- HAProxy image pullPolicy.
     pullPolicy: IfNotPresent
-    # haproxy.image.pullSecrets -- HAProxy repository pullSecret (ex: specify docker registry credentials)
-    ## Ref: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+    # haproxy.image.pullSecrets -- The [imagePullSecrets](https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod)
+    # to reference for the HAProxy Pods.
     pullSecrets: []
-    # haproxy.image.tag -- HAProxy image tag to use
+    # haproxy.image.tag -- The tag to use for HAProxy's image.
     tag: "2.4.17"
 
-  # haproxy.rollWorkload -- Add a checksum of the generated ConfigMap to the HAProxy Deployment
+  # haproxy.rollWorkload -- Add a checksum of the generated ConfigMap to the HAProxy Deployment.
   rollWorkload: true
 
-  # haproxy.replicas -- Set the number of HAProxy Pods to create
+  # haproxy.replicas -- Set the number of HAProxy Pods to create.
   replicas: 1
 
   serviceAccount:
-    # haproxy.serviceAccount.create -- If true, create a HAProxy ServiceAccount
+    # haproxy.serviceAccount.create -- If true, create a HAProxy ServiceAccount.
     create: true
-    # haproxy.serviceAccount.annotations -- Annotations to add to the HAProxy ServiceAccount
+    # haproxy.serviceAccount.annotations -- Annotations to add to the HAProxy ServiceAccount.
     annotations: {}
-    # haproxy.serviceAccount.name -- The name of the HAProxy ServiceAccount to use.
-    ## If not set and create is true, a name is generated using the fullname template
+    # haproxy.serviceAccount.name -- The name of the HAProxy ServiceAccount to use. If not set and create is true, a
+    # name is generated using the fullname template.
```

Comment:
```suggestion
    # haproxy.serviceAccount.name -- The name of the HAProxy ServiceAccount to use. If not set and `create` is true, a
    # name is generated using the fullname template.
```
?

Reply:
We've not done that in other comments, so I'll leave that change for a separate PR.